util/cron/nightly

```diff
@@ -710,6 +714,13 @@ if ($runtests == 0) {
     }
 
     $ENV{'CHPL_HOME'} = $chplhomedir;
+
+    # Confirm Chapel built correctly before start_test / paratest
+    if ($buildcheck == 1) {
+        $buildcheckcommand = "cd $ENV{'CHPL_HOME'} && make check";
+        mysystem($buildcheckcommand, "Running `make check`", 1, 1, 1);
```

Comment:
Also really minor, but I'd lowercase "Running" to "running" since mysystem() will print "Error Running `make check`"